src/pyHepGrid/src/runmodes.py

```diff
@@ -2,4 +2,5 @@
 
```

Comment:
Should we make the mode choice case insensitive by using all caps in this dictionary here? 
i.e. 
```suggestion
"SHERPA: pyHepGrid.src.programs.Sherpa,
```
and then doing header.runmode.upper() when assigning to _mode on line 13 of Backend.py. I think that makes it more resilient and intuitive to the user. 

Other than that, it all looks good (I assume that you've tested that it runs correctly)

Reply:
Done in c49de344c8990b90c39a22f98406aeb01021cd7d